sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -129,5 +129,144 @@ describe(
       });
       expect(scheduleIsSent).toBe(true);
     });
+
+    it("Should create an e-Cert with three disbursements for two different students with two disbursements each where three records are eligible.", async () => {
+      // Arrange
+
+      const coeUpdatedAt = new Date();
+
+      // Student A with valid SIN.
+      const studentA = await saveFakeStudent(db.dataSource);
+      // Valid MSFAA Number.
+      const msfaaNumberA = await db.msfaaNumber.save(
+        createFakeMSFAANumber(
+          { student: studentA },
+          { msfaaState: MSFAAStates.Signed },
```

Comment:
Shouldn't the MSFAA number be part time here?

Reply:
Changed.